ecs-cli/modules/config/readwriter_v1_test.go

```diff
@@ -0,0 +1,341 @@
+
+}
+
+func TestPrefixesEmptyNewYAMLFormat(t *testing.T) {
+	dest, err := newMockDestination()
+	assert.NoError(t, err, "Error creating mock config destination")
+
+	parser := setupParser(t, dest, false)
+
+	err = os.MkdirAll(dest.Path, *dest.Mode)
+	assert.NoError(t, err, "Could not create config directory")
+
+	defer os.RemoveAll(dest.Path)
+
+	saveConfigWithCluster(t, parser, dest)
+
+	// Reinitialize from the written file.
+	parser = setupParser(t, dest, true)
+
+	readConfig, configMap, err := parser.GetConfig()
+	assert.NoError(t, err, "Error reading config")
+	assert.Equal(t, testClusterName, readConfig.Cluster, "Cluster name mismatch in config.")
+	_, ok := configMap[composeProjectNamePrefixKey]
+	assert.True(t, ok, "Compose project prefix name should exist in config.")
+	assert.Empty(t, readConfig.ComposeProjectNamePrefix, "Compose project prefix name should be empty.")
+	_, ok = configMap[composeServiceNamePrefixKey]
+	assert.True(t, ok, "Compose service name prefix should exist in config.")
+	assert.Empty(t, readConfig.ComposeServiceNamePrefix, "Compose service prefix name should be empty.")
+	_, ok = configMap[cfnStackNamePrefixKey]
+	assert.True(t, ok, "CFNStackNamePrefix should exist in config.")
+	assert.Empty(t, readConfig.CFNStackNamePrefix, "CFNStackNamePrefix should be empty.")
+}
+
+func TestPrefixesEmptyOldINIFormat(t *testing.T) {
+	configContents := `[ecs]
+cluster = test-cluster
+aws_profile = testProfile
+region = us-west-2
+aws_access_key_id =
+aws_secret_access_key =
+compose-project-name-prefix =
+compose-service-name-prefix =
+cfn-stack-name-prefix =
+`
+	dest, err := newMockDestination()
+	assert.NoError(t, err, "Error creating mock config destination")
+
+	err = os.MkdirAll(dest.Path, *dest.Mode)
+	assert.NoError(t, err, "Could not create config directory")
+	defer os.RemoveAll(dest.Path)
+
+	err = ioutil.WriteFile(configPath(dest), []byte(configContents), *dest.Mode)
+	assert.NoError(t, err)
+
+	// Reinitialize from the written file.
+	parser := setupParser(t, dest, true)
+
+	readConfig, configMap, err := parser.GetConfig()
+	assert.NoError(t, err, "Error reading config")
+	assert.Equal(t, testClusterName, readConfig.Cluster, "Cluster name mismatch in config.")
+	_, ok := configMap[composeProjectNamePrefixKey]
+	assert.True(t, ok, "Compose project prefix name should exist in config.")
+	assert.Empty(t, readConfig.ComposeProjectNamePrefix, "Compose project prefix name should be empty.")
+	_, ok = configMap[composeServiceNamePrefixKey]
+	assert.True(t, ok, "Compose service name prefix should exist in config.")
+	assert.Empty(t, readConfig.ComposeServiceNamePrefix, "Compose service prefix name should be empty.")
+	_, ok = configMap[cfnStackNamePrefixKey]
+	assert.True(t, ok, "CFNStackNamePrefix should exist in config.")
+	assert.Empty(t, readConfig.CFNStackNamePrefix, "CFNStackNamePrefix should be empty.")
+}
+
+func TestPrefixesDefaultOldINIFormat(t *testing.T) {
```

Comment:
Good catch- this is a mistake on my part. These test cases are supposed to test different things.